core/templates/dev/head/pages/exploration-player-page/services/extract-image-filenames-from-state.service.ts

```diff
@@ -117,9 +117,9 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
      * @param {string} strHtml - The string from which the object of
      *                           filepath should be extracted.
      */
-    var _extractFilepathValueFromOppiaNonInteractiveImageTag = function(
+    var _extractImageIdValueFromOppiaNonInteractiveImageTag = function(
```

Comment:
You would need to rename the file too.

Reply:
Why sir, though there is the renaming of function at the end you see, we are returning filenames.
Did you think there should be renaming? I don't think so!

Reply:
Oh I see. Thanks for the explanation.